shell/key-bindings.fish

```diff
@@ -84,4 +86,26 @@ function fzf_key_bindings
     bind -M insert \cr fzf-history-widget
     bind -M insert \ec fzf-cd-widget
   end
+
+  function __fzf_get_starting_dir
+    set -l dir (commandline -t)
+
+    # Default to current dir if no argument input
+    if [ -z "$dir" ]
+        set dir '.'
+    end
+
+    # The commandline token might be escaped, we need to unescape it.
+    set dir (eval "printf '%s' $dir")
```

Comment:
To clarify, the current behaviour is that `vim $HOME<ctrl-t>` will resolve to `vim /Users/ipwnponies/foo.txt`. This is because the `$dir` input is passed to `find $dir`: if `$dir` is a relative path, the results are relative; otherwise the results are absolute paths.

Another clarification as to why I said this `eval "printf '%s' $dir"` is not required, `find $HOME` (no `eval`) and `find /Users/ipwnponies/` (with `eval`) output the same values and are functionally equivalent.